<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgModeler</class>
 <widget class="QMainWindow" name="DlgModeler">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>891</width>
    <height>596</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Processing Modeler</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_1">
    <property name="spacing">
     <number>3</number>
    </property>
    <property name="leftMargin">
     <number>2</number>
    </property>
    <property name="topMargin">
     <number>2</number>
    </property>
    <property name="rightMargin">
     <number>2</number>
    </property>
    <property name="bottomMargin">
     <number>2</number>
    </property>
    <item>
     <widget class="QGraphicsView" name="view"/>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="mToolbar">
   <property name="windowTitle">
    <string>Navigation</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="mActionOpen"/>
   <addaction name="mActionSave"/>
   <addaction name="mActionSaveAs"/>
   <addaction name="mActionSaveInProject"/>
   <addaction name="separator"/>
   <addaction name="mActionZoomIn"/>
   <addaction name="mActionZoomOut"/>
   <addaction name="mActionZoomActual"/>
   <addaction name="mActionZoomToItems"/>
   <addaction name="separator"/>
   <addaction name="mActionExportImage"/>
   <addaction name="mActionExportPdf"/>
   <addaction name="mActionExportSvg"/>
   <addaction name="separator"/>
   <addaction name="mActionEditHelp"/>
   <addaction name="separator"/>
   <addaction name="mActionRun"/>
  </widget>
  <action name="mActionOpen">
   <property name="text">
    <string>Open model…</string>
   </property>
   <property name="toolTip">
    <string>Open model (Ctrl+O)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="mActionSave">
   <property name="text">
    <string>Save model</string>
   </property>
   <property name="toolTip">
    <string>Save model (Ctrl+S)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="mActionSaveAs">
   <property name="text">
    <string>Save model as…</string>
   </property>
   <property name="toolTip">
    <string>Save model as (Ctrl+S)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="mActionZoomActual">
   <property name="text">
    <string>Zoom to &amp;100%</string>
   </property>
   <property name="toolTip">
    <string>Zoom to &amp;100%</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+1</string>
   </property>
  </action>
  <action name="mActionZoomIn">
   <property name="text">
    <string>Zoom in</string>
   </property>
   <property name="toolTip">
    <string>Zoom in</string>
   </property>
   <property name="shortcut">
    <string>Ctrl++</string>
   </property>
  </action>
  <action name="mActionZoomOut">
   <property name="text">
    <string>Zoom out</string>
   </property>
   <property name="toolTip">
    <string>Zoom out</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+-</string>
   </property>
  </action>
  <action name="mActionExportImage">
   <property name="text">
    <string>Export as image…</string>
   </property>
   <property name="toolTip">
    <string>Export as image</string>
   </property>
  </action>
  <action name="mActionZoomToItems">
   <property name="text">
    <string>Zoom full</string>
   </property>
   <property name="toolTip">
    <string>Zoom full</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+0</string>
   </property>
  </action>
  <action name="mActionExportPdf">
   <property name="text">
    <string>Export as PDF…</string>
   </property>
   <property name="toolTip">
    <string>Export as PDF</string>
   </property>
  </action>
  <action name="mActionExportSvg">
   <property name="text">
    <string>Export as SVG…</string>
   </property>
   <property name="toolTip">
    <string>Export as SVG</string>
   </property>
  </action>
  <action name="mActionExportPython">
   <property name="text">
    <string>Export as Python Script…</string>
   </property>
   <property name="toolTip">
    <string>Export as Python Script</string>
   </property>
  </action>
  <action name="mActionEditHelp">
   <property name="text">
    <string>Edit model help…</string>
   </property>
   <property name="toolTip">
    <string>Edit model help</string>
   </property>
  </action>
  <action name="mActionRun">
   <property name="text">
    <string>Run model…</string>
   </property>
   <property name="toolTip">
    <string>Run model (F5)</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
  </action>
  <action name="mActionSaveInProject">
   <property name="text">
    <string>Save model in project</string>
   </property>
   <property name="toolTip">
    <string>Save model in project</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
